docker-nlp-toolset/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM maluuba/tomcat7
+MAINTAINER Maluuba NLP Team <nlp@maluuba.com>
+
+RUN apt-get install -qqy git docker.io
+
+# Maven
+RUN apt-get remove maven2
+ENV MAVEN_VERSION 3.2.5
+RUN curl -sSL http://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar xzf - -C /usr/share \
+  && mv /usr/share/apache-maven-$MAVEN_VERSION /usr/share/maven \
+  && ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
+ENV MAVEN_HOME /usr/share/maven
+
+RUN mvn -version
+
+# Ruby
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+
+RUN curl -sSL get.rvm.io | bash -s stable
+RUN bash -l -c "rvm requirements"
```

Comment:
I'm hoping @usmanismail knows some way around starting another process.  This is needed for rvm, the normal set up says to log out and log back in.